django/db/migrations/serializer.py

```diff
@@ -270,6 +272,40 @@ def serialize(self):
         return "uuid.%s" % repr(self.value), {"import uuid"}
 
 
+class ClassSerializer(BaseSerializer):
+    def serialize(self):
+        _module = self.value.__module__
+        module_first_part, module_second_part = _module.split('.')
+        _class = self.value.__class__
+        _path = ""
+        if module_second_part.startswith('_'):
```

Comment:
This code is highly specific to make the `psycopg2.extras._range.DateTimeTZRange` case pass tests.

What about non-leaf level mode private imports such as `package._private.module` or undocumented modules that shouldn't be directly imported from such as  `django.db.models.fields`?